src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -192,34 +185,34 @@ public void add(final ScriptInfo other) {
 			commands += other.commands;
 			functions += other.functions;
 		}
-		
+
 		public void subtract(final ScriptInfo other) {
 			files -= other.files;
 			triggers -= other.triggers;
 			commands -= other.commands;
 			functions -= other.functions;
 		}
-		
+
 		@Override
 		public String toString() {
 			return "ScriptInfo{files=" + files + ",triggers=" + triggers + ",commands=" + commands + ",functions:" + functions + "}";
 		}
 	}
-	
+
 //	private final static class SerializedScript {
 //		public SerializedScript() {}
 //
 //		public final List<Trigger> triggers = new ArrayList<Trigger>();
 //		public final List<ScriptCommand> commands = new ArrayList<ScriptCommand>();
 //	}
-	
+
 	private static String indentation = "";
-	
+
 	// Load scripts in separate (one) thread
 	static final BlockingQueue<Runnable> loadQueue = new ArrayBlockingQueue<>(20, true);
-	static final Thread loaderThread;
+	private static final Thread loaderThread;

```

Comment:
Asynchronous loading is not entirely finished. This might be needed from elsewhere.